tests/app/parser/test_metadata_parser.py

```diff
@@ -134,13 +142,13 @@ def test_missing_required_metadata_user_id_in_token(self):
         with self.assertRaises(InvalidTokenException) as ite_key:
             validate_metadata(metadata, self.schema_metadata)
 
-        self.assertEqual('Missing key/value for user_id', str(ite_key.exception))
+        self.assertEqual('incorrect data in token', str(ite_key.exception))
```

Comment:
It would be good to keep the existing error message as it was very helpful the other day when trying to see which item of the claims was preventing RH from launching